### DISCUSSION TITLE
App is crashing after adding react-hook-form

### DISCUSSION THREAD

**Discussion by arstiefel** [react-hook-form/react-hook-form#8051] [Q&A] [answered]
We have a very large form that we're trying to migrate over to react-hook-form. I am doing it step by step by converting class components to FCs and adding the form. I'm using the example https://react-hook-form.com/advanced-usage#ConnectForm to connect my form in my various components. However, when I try to run my application it crashes on load with the following

```
node:events:368
      throw er; // Unhandled 'error' event
      ^

Error: read ECONNRESET
    at TCP.onStreamRead (node:internal/stream_base_commons:220:20)
Emitted 'error' event on Socket instance at:
    at emitErrorNT (node:internal/streams/destroy:157:8)
    at emitErrorCloseNT (node:internal/streams/destroy:122:3)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  errno: -4077,
  code: 'ECONNRESET',
  syscall: 'read'
}

```

There's no information in any log. Has anyone seen this behavior?

**Comment by Moshyfawn** [comment, marked as answer]
Looks like a Node related issue, not RHF, TBH.

You'll have to provide some minimal repro or seek the solution with the Node community instead, I'm afraid.

What Node version is your app using?

**Reply by arstiefel** [reply]
This ended up getting resolved. Not sure what happened

**Reply by Moshyfawn** [reply]
Heh, well, I'm glad it's resolved :P
